### DISCUSSION TITLE
How can I split the tests among different machines

### DISCUSSION THREAD

**Discussion by arvinder06** [VinayKumarBM/playwright-cucumber-sample#2] [Q&A] [answered]
Hi Vinay,

I was going through your work, and I really liked it. However, I am wondering how we can split the workload among different machines? I am planning to run cucumber js playwright tests on Kubernetes cluster and just not sure how can I split the tests among different pod. I mean, I can do it using just the playwright runner (sharding functionality put of the box), however, I am not sure how can this be achieve using cucumber js. Any idea please, as I really want to achieve this by using BDD cucumber.

Thanks

**Comment by VinayKumarBM** [comment, marked as answer]
What you are asking for is a feature of Playwright Test Runner, we are using Cucumber to run test here and I think it does not have this feature as of now.
We only have option to run the scenarios parallelly.
